manifests/n/NetEase/Minecraft/1.17.5.8/NetEase.Minecraft.installer.yaml

```
@@ -0,0 +1,27 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: NetEase.Minecraft
+PackageVersion: 1.17.5.8
+InstallerLocale: zh-CN
+InstallerType: inno
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'
+ReleaseDate: 2024-12-17
+#AppsAndFeaturesEntries: --> By komac
+#- DisplayName: 网易发烧游戏
+#  Publisher: Netease
+#  ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'

```

```suggestion
AppsAndFeaturesEntries:
- DisplayName: 网易发烧游戏
  Publisher: Netease
  ProductCode: '{17EF2B1F-E35E-414E-850A-D45E7D4717F4}_is1'
```
Maybe komac is correct...